### PR TITLE
SUPMWS-273 improve error handling

### PR DESCRIPTION
**Pull request by sfozz**
removed the use of `debug()` and put in place winston (generic logging) and morgan (Access log)

### CODE REVIEW

File: lib/actions.js [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
     res.send("PONG");
   },
   sendFile(req, res) {
-    debug("send file");
+    req.app.U.logger.debug("in sendFile");
     const readStream = fs.createReadStream(`${req.temp.name}/output.pdf`);
+    let responseSent = false;
+    setTimeout(() => {
+      if (!responseSent) {
+        req.app.U.logger.error("PDF not sent");
```
Comment (markstuart) [inline lib/actions.js:15]: Do we want to `req.temp.removeCallback()` here? This scenario is that the pdf generation succeeded but the request that triggered the generation has gone away, probably can delete the temp files?

File: lib/actions.js [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
-    readStream.on("end", () => req.temp.removeCallback());
-    readStream.on("error", (err) => res.status(500).send(err));
+    readStream.on("end", () => {
+      req.temp.removeCallback();
+    });
+    readStream.on("close", () => {
+      req.app.U.logger.debug("in close");
+    });
+    readStream.on("error", (err) => {
+      req.app.U.logger.error("there was and issue sending the pdf", err);
```
Comment (markstuart) [inline lib/actions.js]: "There was **an** issue"

File: README.md [hunk 3]
```diff
@@ -54,6 +54,6 @@ This module can of course be run locally without Docker, however you will need t
 - **wkhtmltopdf**
 - **pdftk**
 
-After pulling down this repo and running `npm install` you can start the service using default settings with `node .`, there are currently two optional args `--port` and `--limit` for setting the port to listen on and maximum allowed body size for the request.
+After pulling down this repo and running `npm install` you can start the service using default settings with `node .`, there are currently there optional args `--port`, `--limit`, and `--loglevel` for setting the port to listen on, maximum allowed body size for the request, and the log level.
```
Comment (tim-mediasuite) [inline README.md]: typo, three

**Review by tim-mediasuite — APPROVED** [review]
Looks good

**Pull request merged**